src/components/KSlideout/KSlideout.vue

```diff
@@ -77,18 +90,10 @@ onUnmounted(() => {
   --KCardPaddingY: 32px;
   --KCardPaddingX: 48px;
 
-  .panel-background {
-    background: var(--black-45, color(black-45));
-    bottom: 0;
-    left: 0;
-    position: fixed;
-    right: 0;
-    top: 0;
-    z-index: 9999;
-  }
-
   .panel {
     background-color: var(--white, color(white));
+    display: flex;
+    flex-direction: column;
```

Comment:
question: would it make sense to add a new prop `topOffset<number>` that allows for modifying the CSS `top: 0` rule below? It could default to `0`, but would allow a host app to define the offset from the top of the page.

In Konnect UI, we could pass in the height of the navbar and display the sidebar under the nav
![image](https://github.com/Kong/kongponents/assets/2229946/3bb90904-98fd-40c2-8ef8-2cd2bcd6b7fe)

Reply:
Done